force-app/main/default/lwc/apexWireMethodWithParams/__tests__/apexWireMethodWithParams.test.js

```diff
@@ -0,0 +1,85 @@
+import { createElement } from 'lwc';
+import ApexWireMethodWithParams from 'c/apexWireMethodWithParams';
+import { registerLdsTestWireAdapter } from '@salesforce/wire-service-jest-util';
+import findContacts from '@salesforce/apex/ContactController.findContacts';
+
+// Realistic data with a list of contacts
+const mockFindContacts = require('./data/findContacts.json');
+// An empty list of records to verify the component does something reasonable
+// when there is no data to display
+const mockFindContactsNoRecords = require('./data/findContactsNoRecords.json');
+
+// Register as an LDS wire adapter. Some tests verify that provisioned values trigger desired behavior.
+const findContactsAdapter = registerLdsTestWireAdapter(findContacts);
+
+describe('c-apex-wire-method-with-params', () => {
+    beforeAll(() => {
+        jest.useFakeTimers();
+    });
+
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    describe('findContacts @wire data', () => {
+        it('with one record', () => {
```

Comment:
What is the behavior you're verifying here? `findContacts @wire data > with one record` explains the scenario you're executing but not what you're expecting to happen.

Reply:
Description changed